spec/scss_lint/linter/url_format_spec.rb

```diff
@@ -31,6 +31,12 @@
 
       it { should_not report_lint }
     end
+
+    context 'when URL contains a variable' do
+      let(:url) { '${' }
```

Comment:
This would be better if it had the full string, e.g. `'${some-var}'`. I know that your code doesn't care, but I could see someone changing the check to a more involved regex matching the whole variable inject.